pandas/tests/io/formats/test_format.py

```diff
@@ -541,7 +541,7 @@ def test_to_string_truncate(self):
         df.set_index(["a", "b", "c"])
         assert df.to_string() == (
             "     a    b                                         "
-            "                                                c  d\n"
+            "                                                c  d\n" # how to change?
```

Comment:
@datapythonista how would you change these lines? (if you even change them)

This is the last case (CI perspective).
If you wouldn't change these lines what will you say be a good ```if``` statement for the CI?

Reply:
I think they look good, and they shouldn't fail the CI with what we discussed, since we don't validate cases where there is more than one space. The error in the CI is because you're missing one spaces before the inline comment `# how to change?`. I think everything should be correct once you remove that comment.

Am I missing something?

Reply:
Got it!
Will fix the CI :)

Thank you @datapythonista